internal/provisioner/kops_provisioner.go

```diff
@@ -224,11 +225,52 @@ func (provisioner *KopsProvisioner) CreateCluster(cluster *model.Cluster) error
 		}
 	}
 
+	// Setup Helm
+	err = HelmSetup(logger, kops)
+	if err != nil {
+		return err
+	}
+
 	logger.WithField("name", kopsMetadata.Name).Info("Successfully created cluster")
 
 	return nil
 }
 
+// HelmSetup is used for the initial setup of Helm in cluster
+func HelmSetup(logger log.FieldLogger, kops *kops.Cmd) error {
+	logger.Infof("Initializing Helm in the cluster")
+	init := exec.Command("helm", "--kubeconfig", kops.GetKubeConfigPath(), "init", "--upgrade")
```

Comment:
you dont need to init here, you can create the service account  and roles and in the end apply the helm init with the service account